src/dialects/abstract/query-interface.js

```diff
@@ -1066,10 +1119,14 @@ export class QueryInterface {
 
     const dataType = options.dataType;
 
+    // FIXME: DECIMAL is not safely representable as a float!
```

Comment:
This is for a next PR right? And why did you use FIXME here instead of TODO?

Reply:
For another PR yep, it's just a thing I noticed while I was working on that file
I'll change it to TODO for consistency. I sometimes use `FIXME` when it's a bug but it's probably a bad idea, `TODO` is universally recognized, `FIXME` isn't